.travis.yml

```diff
@@ -28,12 +28,18 @@ env:
   - DH_REPO="cloudsuite/spark" IMG_TAG="1.5.1" DF_PATH="./commons/spark/1.5.1"
   - DH_REPO="cloudsuite/spark" IMG_TAG="2.1.0" DF_PATH="./commons/spark/2.1.0"
 
+  # Faban
+  - DH_REPO="cloudsuite/faban" IMG_TAG="1.3" DF_PATH="./commons/faban/1.3"
+  - DH_REPO="cloudsuite/faban" IMG_TAG="1.4" DF_PATH="./commons/faban/1.4"
+  - DH_REPO="cloudsuite/faban" IMG_TAG="1.4.1" DF_PATH="./commons/faban/1.4.1"
```

Comment:
Could you also add a `latest` tag?
I think we should either have `1.4` or `1.4.1`, it doesn't make sense to have both.